.gitignore

```diff
@@ -1,5 +1,8 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+# lockfile from npm 
+package-lock.json
```

Comment:
Also an unrelated change deleted the lockfile and added it to the ignore
file since we use pnpm.